components/index/cards/communityHub.js

```
@@ -51,70 +53,97 @@ export default function Onboard({ stars }) {
-          <Text as="p" variant="subtitle" sx={{ mt: [3, 4, 5], color: 'white' }}>
-            Circuit boards are magical. You design one, we'll print it.
-            Completely for free! Get a $100 grant to fuel the creation of your
-            dream project with OnBoard.
-          </Text>
-        </Box>
-        <Flex
-          sx={{ flexDirection: 'column', mt: [4, 5, 5],  ml: [0, 2, 3], placeSelf: 'start' }}
+        <source src="/home/Communityhub1.mp4" type="video/mp4" />
+      </video>
+      
+      /* Mobile video - visible only on mobile */
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)